invenio/b2share/modules/b2deposit/b2share_marc_handler.py

```diff
@@ -157,6 +158,12 @@ def add_file_info(rec, form, email, sub_id, recid):
     else:
         fft_status = 'firerole: allow email "{0}"\ndeny all'.format(
             email)
+
+    if 'embargo_till' in form:
+        embargodate = parser.parse(form['embargo_till'])
+        embargodate = datetime.strftime(embargodate, '%Y-%m-%d')
+        fft_status = 'firerole: deny until "%s"\nallow any\n' % embargodate
```

Comment:
It seemed to me that `deny until` is equivalent to `allow email` until end of embargo. The way I wrote it seems to be how it has been done elsewhere.

I had to re-build my VM, so have not tested all use cases myself yet.